grammar.js

```diff
@@ -201,19 +203,19 @@ module.exports = grammar({
       $._lhs
     ),
 
-    scope_resolution_expression: $ => prec.left(seq(optional($._primary), '::', $.identifier)),
-    element_reference: $ => prec.left(seq($._primary, "[", $._argument_list, "]")),
-    member_access: $ => prec.left(seq($._primary, ".", $.identifier)),
+    scope_resolution_expression: $ => prec.left(1, seq(optional($._primary), '::', $.identifier)),
```

Comment:
I’m not familiar with `prec.left(1, ...)` and `prec.left(0, ...)`, but it seems like we use that in a few places. would it be useful to break that out into a function?